server/views/partials/header.njk

```diff
@@ -13,26 +13,28 @@
       <a class="hmpps-header__link hmpps-header__title__service-name" href="/">Digital Prison Services</a>
     </div>
 
+
     <nav aria-label="Account navigation">
       <ul class="hmpps-header__navigation">
-        {% if caseLoads.length > 1 %}
-          <li class="hmpps-header__navigation__item">
-            <img class="hmpps-header__link__icon govuk-!-padding-right-3" src="{{ ingressUrl }}/assets/images/location.svg" aria-hidden="true" alt="" />
-            <a data-qa="changeCaseLoad" class="hmpps-header__link" href="{{ changeCaseLoadLink }}" data-test="change-case-load-link" target="_blank" rel="noopener" referrerpolicy="no-referrer-when-downgrade">
-              <span data-qa=header-active-case-load>{{ activeCaseLoad.description }}</span>
-            </a>
-          </li>
-        {% endif %}
+        {% if isPrisonUser === true %}
+            {% if caseLoads.length > 1 %}
+              <li class="hmpps-header__navigation__item">
+                <img class="hmpps-header__link__icon govuk-!-padding-right-3" src="{{ ingressUrl }}/assets/images/location.svg" aria-hidden="true" alt="" />
+                <a data-qa="changeCaseLoad" class="hmpps-header__link" href="{{ changeCaseLoadLink }}" data-test="change-case-load-link" target="_blank" rel="noopener" referrerpolicy="no-referrer-when-downgrade">
+                  <span data-qa=header-active-case-load>{{ activeCaseLoad.description }}</span>
+                </a>
+              </li>
+            {% endif %}
 
-        {% if user %}
-          <li class="hmpps-header__navigation__item">
-            <a data-qa="manageDetails" class="hmpps-header__link" href="/account-details" data-test="manage-account-link" target="_blank" rel="noopener noreferrer">
-              <span data-qa=header-user-name>{{ user.displayName | initialiseName }}</span>
-              <span class="hmpps-header__link__sub-text">Manage your details</span>
-            </a>
-          </li>
+            {% if user %}
+              <li class="hmpps-header__navigation__item">
+                <a data-qa="manageDetails" class="hmpps-header__link" href="/account-details" data-test="manage-account-link" target="_blank" rel="noopener noreferrer">
```

Comment:
I don't think the manage account page should open in a new tab/window - the current one doesn't